UICatalog/Scenarios/ListColumns.cs

```diff
@@ -0,0 +1,332 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Data;
+using Terminal.Gui;
+using static Terminal.Gui.TableView;
+
+namespace UICatalog.Scenarios {
+
+	[ScenarioMetadata (Name: "ListColumns", Description: "Implements a columned list via a data table.")]
+	[ScenarioCategory ("TableView")]
+	[ScenarioCategory ("Controls")]
+	[ScenarioCategory ("Dialogs")]
+	[ScenarioCategory ("Text and Formatting")]
+	[ScenarioCategory ("Top Level Windows")]
+	public class ListColumns : Scenario {
+		TableView listColView;
+		DataTable currentTable;
+		private MenuItem miCellLines;
```

Comment:
Prefix with `_`

Reply:
This was a copy-paste from TableEditor scenario.  Should I change both?

Reply:
I would say only change in the new code.  

If we want to update old code to have this notation it should be a separate PR.

Reply:
Yeah, this (and the scrollbar one mentioned below) affect several Scenarios.  I'll do a separate PR.